src/test/integration/IntegrationChecks.t.sol

```diff
@@ -104,8 +104,8 @@ contract IntegrationCheckUtils is IntegrationBase {
 
         assert_Snap_Unchanged_Staker_DepositShares(staker, "staker shares should not have decreased");
         assert_Snap_Removed_Staker_WithdrawableShares_AtLeast(staker, BEACONCHAIN_ETH_STRAT, slashedAmountGwei * GWEI_TO_WEI, "should have decreased withdrawable shares by at least slashed amount");
-        assert_Snap_Removed_ActiveValidatorCount(staker, slashedValidators.length, "should have decreased active validator count");
-        assert_Snap_Removed_ActiveValidators(staker, slashedValidators, "exited validators should each be WITHDRAWN");
+        //assert_Snap_Removed_ActiveValidatorCount(staker, slashedValidators.length, "should have decreased active validator count");
+        //assert_Snap_Removed_ActiveValidators(staker, slashedValidators, "exited validators should each be WITHDRAWN");
```

Comment:
What's up with these?

Reply:
Okay, figured out what's wrong with these. TLDR this assertion is only used when we do a slash that doesn't withdraw validators. I added a clarifying comment here [86c255d](https://github.com/Layr-Labs/eigenlayer-contracts/pull/1045/commits/86c255d601f1d7c96dc7428c8f2a3aee47ba5f0b)